src/routes/InstancesPage/InstancesToolbarSearchFilter.js

```diff
@@ -49,13 +55,17 @@ function InstancesToolbarSearchFilter({ filters, setFilters }) {
   function onSelect(type, event, selection) {
     const checked = event.target.checked;
     setFilters((prevFilters) => {
+      const newFilters = { ...prevFilters };
       const prevSelections = prevFilters[type] || [];
-      return {
-        ...prevFilters,
-        [type]: checked
-          ? [...prevSelections, selection]
-          : prevSelections.filter((value) => value !== selection),
-      };
+      const newValue = checked
+        ? [...prevSelections, selection]
+        : prevSelections.filter((value) => value !== selection);
+      if (newValue?.length === 0) {
```

Comment:
Nit: you can drop the optional chain `?` here I think since `prevSelections` will always be an array.